From 993c8b6ca80e0aa12290262df52ed36f10e33796 Mon Sep 17 00:00:00 2001
From: Markuss Broks <markuss.broks@gmail.com>
Date: Sat, 22 Aug 2026 00:30:13 +0300
Subject: [PATCH 03/37] dt-bindings: input: touchscreen: imagis: add compatible
 for IST4050

The IST4050 is a touchscreen IC of the Imagis IST40xx family. It shares
the register interface with IST3038C, but uses a different, event-based
touch reporting format and does not have touch keys.

The binding no longer covers only the IST30XXC family, so generalize its
title as well.

Signed-off-by: Markuss Broks <markuss.broks@gmail.com>
---
 .../devicetree/bindings/input/touchscreen/imagis,ist3038c.yaml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/input/touchscreen/imagis,ist3038c.yaml b/Documentation/devicetree/bindings/input/touchscreen/imagis,ist3038c.yaml
index dfaffbc39..a4f3e2f0c 100644
--- a/Documentation/devicetree/bindings/input/touchscreen/imagis,ist3038c.yaml
+++ b/Documentation/devicetree/bindings/input/touchscreen/imagis,ist3038c.yaml
@@ -4,7 +4,7 @@
 $id: http://devicetree.org/schemas/input/touchscreen/imagis,ist3038c.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: Imagis IST30XXC family touchscreen controller
+title: Imagis touchscreen controller
 
 maintainers:
   - Markuss Broks <markuss.broks@gmail.com>
@@ -20,6 +20,7 @@ properties:
       - imagis,ist3038b
       - imagis,ist3038c
       - imagis,ist3038h
+      - imagis,ist4050
 
   reg:
     maxItems: 1
